docs/en/03_Upgrading/index.md

```diff
@@ -1,22 +1,25 @@
 title: Upgrading
 introduction: Keep your SilverStripe installations up to date with the latest fixes, security patches and new features.
 
-# Upgrading
+# Upgrading to SilverStripe 3.2
```

Comment:
This section is going to get stale very fast. It needs to be generic, otherwise we'll have to constantly update this section each release.